test/integration/subscription_test.exs

```diff
@@ -6,9 +6,25 @@ defmodule Braintree.Integration.SubscriptionTest do
   alias Braintree.Customer
   alias Braintree.Subscription
 
+  def create_test_subscription do
+    {:ok, customer} = Customer.create(%{payment_method_nonce: "fake-valid-nonce"})
+    [card] = customer.credit_cards
+    Subscription.create(%{payment_method_token: card.token, plan_id: "starter"})
+  end
+
   test "create/1 with a plan_id" do
     assert {:ok, customer} = Customer.create(%{payment_method_nonce: "fake-valid-nonce"})
     [card] = customer.credit_cards
-    assert {:ok, _subscription } = Subscription.create(%{payment_method_token: card.token, plan_id: "starter"})
+    assert {:ok, _subscription} = Subscription.create(%{payment_method_token: card.token, plan_id: "starter"})
+  end
+
+  test "find/1 with a subscription_id" do
+    {:ok, subscription} = create_test_subscription
+    assert {:ok, _subscription} = Subscription.find(subscription.id)
+  end
+
+  test "cancel/1 with a subscription_id" do
+    {:ok, subscription} = create_test_subscription
+    assert {:ok, _subscription} = Subscription.cancel(subscription.id)
```

Comment:
Similarly, it would be good to assert something about the canceled subscription. Which property changes? A struct is still returned, there is probably an important property of it.